manager/controlapi/service.go

```diff
@@ -145,39 +210,60 @@ func validateEndpointSpec(epSpec *api.EndpointSpec) error {
 		return nil
 	}
 
+	var me MultiError
+
+	if epSpec.Mode != api.ResolutionModeDNSRoundRobin && epSpec.Mode != api.ResolutionModeVirtualIP {
+		err := grpc.Errorf(codes.InvalidArgument, "EndpointSpec: mode should be in range 0-1")
```

Comment:
I'd prefer not to mention protobuf field numbers in error messages. They won't be meaningful to the user.